部分功能的改进和增添

Just-Prog opened the pull request.

Just-Prog committed:
添加了纸质邮寄票的地址选择功能（待加强测试）以及抢票成功后滑块页跳转订单中心的逻辑

Just-Prog committed:
Update README.md

Just-Prog commented:
有点糊但是确实能用

Just-Prog committed:
修一下requirements.txt

Just-Prog committed:
修改了发生错误退出程序的方法以便于pyinstaller正常打包exe并运行

axiliya2568 commented:
这个只能适用于包邮的，没有包邮的票到抢购的时候会没有加上邮费导致无法购买

Just-Prog commented:
> 这个只能适用于包邮的，没有包邮的票到抢购的时候会没有加上邮费导致无法购买

可以劳烦您提供一下非包邮票据的购买链接吗？我这边所有测试的样本都只有包邮的，我看下能不能这两天有时间修一下

axiliya2568 commented:
厦门·维也纳约翰·施特劳斯经典音乐&世界名曲新年交响音乐会-bilibili会员购漫展票务，这个的

从 Windows 版邮件发送

发件人: Just Prog
发送时间: 2023年12月17日 7:58
收件人: fengx1a0/Bilibili_show_ticket_auto_order
抄送: axiliya2568; Comment
主题: Re: [fengx1a0/Bilibili_show_ticket_auto_order] 纸质票相关功能补充（地址选择） (PR #95)

这个只能适用于包邮的，没有包邮的票到抢购的时候会没有加上邮费导致无法购买
可以劳烦您提供一下非包邮票据的购买链接吗？我这边所有测试的样本都只有包邮的，我看下能不能这两天有时间修一下
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>

axiliya2568 commented:
> > 这个只能适用于包邮的，没有包邮的票到抢购的时候会没有加上邮费导致无法购买
> 
> 可以劳烦您提供一下非包邮票据的购买链接吗？我这边所有测试的样本都只有包邮的，我看下能不能这两天有时间修一下

https://show.bilibili.com/platform/detail.html?id=77749&from=pc_search 这个滴，谢谢大佬了

Just-Prog committed:
修复非包邮票购买请求提交 ( Thanks to @axiliya2568 )

Just-Prog commented:
> axiliya2568

已经测试通过。

![image](https://github.com/fengx1a0/Bilibili_show_ticket_auto_order/assets/62735122/89f71a60-c391-4ce1-8ba6-f13e0034ac32)

axiliya2568 commented:
> > axiliya2568
> 
> 已经测试通过。
> 
> ![image](https://private-user-images.githubusercontent.com/62735122/291071315-89f71a60-c391-4ce1-8ba6-f13e0034ac32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI4MjcwMTcsIm5iZiI6MTcwMjgyNjcxNywicGF0aCI6Ii82MjczNTEyMi8yOTEwNzEzMTUtODlmNzFhNjAtYzM5MS00Y2UxLThiYTYtZjEzZTAwMzRhYzMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE3VDE1MjUxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkZmRhNWRjMWZmMjE4YmNkYzA5ODVmN2QxZmJlZGQzN2M2NWIzMWQ1YTZlN2U3MDJhMTM0MGY1YThmYjVjMDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.e4J8oo1lyBSFU-Xr1xLXqg2d33ZEuZ4Gbe9zfhHM6FA)

大佬厉害，打包的能再更新下吗

axiliya2568 commented:
> > axiliya2568
> 
> 已经测试通过。
> 
> ![image](https://private-user-images.githubusercontent.com/62735122/291071315-89f71a60-c391-4ce1-8ba6-f13e0034ac32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI4Mjc1MzYsIm5iZiI6MTcwMjgyNzIzNiwicGF0aCI6Ii82MjczNTEyMi8yOTEwNzEzMTUtODlmNzFhNjAtYzM5MS00Y2UxLThiYTYtZjEzZTAwMzRhYzMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE3VDE1MzM1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlODAxZTNmOTA5MmUwYWY2M2ZmMTMyMTFmMjJiYWUyYjNhODM1ZTkxOTBjYzBlYTAwZGY4Zjg3OWQxOGE1NTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8gj55raN9FuK-u1_3wa_ExBy-gBgnDEETYi6iXhubTQ)

嘉心糖🐂

axiliya2568 commented:
> > axiliya2568
> 
> 已经测试通过。
> 
> ![image](https://private-user-images.githubusercontent.com/62735122/291071315-89f71a60-c391-4ce1-8ba6-f13e0034ac32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI4OTA4MzcsIm5iZiI6MTcwMjg5MDUzNywicGF0aCI6Ii82MjczNTEyMi8yOTEwNzEzMTUtODlmNzFhNjAtYzM5MS00Y2UxLThiYTYtZjEzZTAwMzRhYzMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE4VDA5MDg1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhMTM0OGFiODYwNjViNmMyOWZhMmNlN2M3ZjgyNjg3ZDA1MGEyNjgyZDZkZjRhNzljOWY5MTU3YTY4YWI3MTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hbuS2IY_VluMHQkoiQNFDr-idIH5Pgr3lWHyyHrP8xI)

大佬还是有点小问题，当选择两张时，不能判断邮费还是按一张的来，导致提示票价发生变化

Just-Prog committed:
fix: 多票包邮统计邮费出错

Just-Prog commented:
> > > axiliya2568
> > 
> > 
> > 已经测试通过。
> > ![image](https://private-user-images.githubusercontent.com/62735122/291071315-89f71a60-c391-4ce1-8ba6-f13e0034ac32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI4OTA4MzcsIm5iZiI6MTcwMjg5MDUzNywicGF0aCI6Ii82MjczNTEyMi8yOTEwNzEzMTUtODlmNzFhNjAtYzM5MS00Y2UxLThiYTYtZjEzZTAwMzRhYzMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE4VDA5MDg1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhMTM0OGFiODYwNjViNmMyOWZhMmNlN2M3ZjgyNjg3ZDA1MGEyNjgyZDZkZjRhNzljOWY5MTU3YTY4YWI3MTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hbuS2IY_VluMHQkoiQNFDr-idIH5Pgr3lWHyyHrP8xI)
> 
> 大佬还是有点小问题，当选择两张时，不能判断邮费还是按一张的来，导致提示票价发生变化

已修

axiliya2568 commented:
> > > > axiliya2568
> > > 
> > > 
> > > 已经测试通过。
> > > ![image](https://private-user-images.githubusercontent.com/62735122/291071315-89f71a60-c391-4ce1-8ba6-f13e0034ac32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI4OTA4MzcsIm5iZiI6MTcwMjg5MDUzNywicGF0aCI6Ii82MjczNTEyMi8yOTEwNzEzMTUtODlmNzFhNjAtYzM5MS00Y2UxLThiYTYtZjEzZTAwMzRhYzMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE4VDA5MDg1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhMTM0OGFiODYwNjViNmMyOWZhMmNlN2M3ZjgyNjg3ZDA1MGEyNjgyZDZkZjRhNzljOWY5MTU3YTY4YWI3MTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hbuS2IY_VluMHQkoiQNFDr-idIH5Pgr3lWHyyHrP8xI)
> > 
> > 
> > 大佬还是有点小问题，当选择两张时，不能判断邮费还是按一张的来，导致提示票价发生变化
> 
> 已修

可以使用🐂

Just-Prog committed:
fix: auth_type error in keyword `一人一票`
逐个词汇鉴别购买人限制感觉不是啥好主意……有啥别的方案吗？

Just-Prog committed:
update .gitignore

Just-Prog committed:
fix: 一人一证条件下购票人数量缺乏判断

Just-Prog committed:
修改了pushpush平台的推送文案和增添了一部分时间输出以便确认购票成功时间
目前测试输出正常（不需要实名认证的票暂时无法在推送文案里带购票人名字）

Luvian77 commented:
上大当，写完才发现已经有pr了
但是没merge，这次Asoul线下没抢到svip😭😭😭

Just-Prog commented:
> 上大当，写完才发现已经有pr了 但是没merge，这次Asoul线下没抢到svip😭😭😭

作者差不多半年多没管这个项目了，估计是bw用完就走了？因为他太长时间没合pr我就给关了

~~顺便一提你怎么知道我帮两个人秒到了svip~~（但是有一个被降级到普通票了不知道咋回事）

Just-Prog commented:
> 上大当，写完才发现已经有pr了 但是没merge，这次Asoul线下没抢到svip😭😭😭

不过我写的功能的细节方面还是跟您有差距，您的相对更赏心悦目一点，我的实现有点太“缝缝补补”

比如我压根没做地址输入超出预期值后的处理方案

axiliya2568 commented:
> > 上大当，写完才发现已经有pr了 但是没merge，这次Asoul线下没抢到svip😭😭😭
> 
> 作者差不多半年多没管这个项目了，估计是bw用完就走了？因为他太长时间没合pr我就给关了
> 
> ~顺便一提你怎么知道我帮两个人秒到了svip~（但是有一个被降级到普通票了不知道咋回事）

 老哥你用的时候不会提示请求频率太快吗，我这提示这个，然后差不多几秒就被提示说太多访问被封了，后面10分才捡漏到svip的，我频率0.5是太快了吗┭┮﹏┭┮

Luvian77 commented:
> > 上大当，写完才发现已经有pr了 但是没merge，这次Asoul线下没抢到svip😭😭😭
> 
> 不过我写的功能的细节方面还是跟您有差距，您的相对更赏心悦目一点，我的实现有点太“缝缝补补”
> 
> 比如我压根没做地址输入超出预期值后的处理方案

我的catch是抄的，还是你的功能多😆商业互吹😆😆😆
我看看能不能找作者来merge一下

Luvian77 commented:
> > > 上大当，写完才发现已经有pr了 但是没merge，这次Asoul线下没抢到svip😭😭😭
> > 
> > 
> > 作者差不多半年多没管这个项目了，估计是bw用完就走了？因为他太长时间没合pr我就给关了
> > ~顺便一提你怎么知道我帮两个人秒到了svip~（但是有一个被降级到普通票了不知道咋回事）
> 
> 老哥你用的时候不会提示请求频率太快吗，我这提示这个，然后差不多几秒就被提示说太多访问被封了，后面10分才捡漏到svip的，我频率0.5是太快了吗┭┮﹏┭┮

是这样的，间隔大一点，或者搞个VPN切代理

Just-Prog commented:
> > > 上大当，写完才发现已经有pr了 但是没merge，这次Asoul线下没抢到svip😭😭😭
> > 
> > 
> > 作者差不多半年多没管这个项目了，估计是bw用完就走了？因为他太长时间没合pr我就给关了
> > ~顺便一提你怎么知道我帮两个人秒到了svip~（但是有一个被降级到普通票了不知道咋回事）
> 
> 老哥你用的时候不会提示请求频率太快吗，我这提示这个，然后差不多几秒就被提示说太多访问被封了，后面10分才捡漏到svip的，我频率0.5是太快了吗┭┮﹏┭┮

有一说一0.5其实在我将近半年的三次大测试中不算快，0.01都试过，还是要看时间段和某些情况，当然如果说同时挂了好几个账号并行抢票的话还是调大点比较好，或者说有能力的话用节点池或者中国大陆其他地域的代理服务器连接

不过我以前碰见过一个很奇葩的网络环境（N级经销商在公众号之类的地方经营的二道贩子宽带），三网都接入，ip一会一变，各个平台定位基本就是两省反复横跳，可以算是抢票圣体

Just-Prog commented:
> > > 上大当，写完才发现已经有pr了 但是没merge，这次Asoul线下没抢到svip😭😭😭
> > 
> > 
> > 作者差不多半年多没管这个项目了，估计是bw用完就走了？因为他太长时间没合pr我就给关了
> > ~顺便一提你怎么知道我帮两个人秒到了svip~（但是有一个被降级到普通票了不知道咋回事）
> 
> 老哥你用的时候不会提示请求频率太快吗，我这提示这个，然后差不多几秒就被提示说太多访问被封了，后面10分才捡漏到svip的，我频率0.5是太快了吗┭┮﹏┭┮

有一说一0.5其实在我将近半年的三次大测试中不算快，0.01都试过，还是要看时间段和某些情况，当然如果说同时挂了好几个账号并行抢票的话还是调大点比较好，或者说有能力的话用节点池或者中国大陆其他地域的代理服务器连接

不过我以前碰见过一个很奇葩的网络环境（N级经销商在公众号之类的地方经营的二道贩子宽带），三网都接入，ip一会一变，各个平台定位基本就是两省反复横跳，可以算是抢票圣体

Just-Prog committed:
针对霹雳霹雳新出的鬼验证码糊了一个粗略的解决方案
感谢群友的点子
感谢 https://kuresaru.github.io/geetest-validator/ 提供校验码获取界面
但愿能用

Just-Prog commented:
浅糊了一个新验证码的方案，实现极其糊，但愿能用

Just-Prog committed:
修复无法下任何单的bug / 修复快递运费到付检测

Just-Prog committed:
文案修改
不发release

Just-Prog committed:
test: 尝试靠拼接seccode的方式节约一点时间

貌似就是多个了这玩意。。。？

KitschEmpathy committed:
feat:自动保存validate在剪贴板

KitschEmpathy committed:
feat:验证成功直接返回validate值

Just-Prog committed:
Merge pull request #2 from KitschEmpathy/main

优化了验证码页面操作流程

Just-Prog committed:
feat:添加自动生成付款二维码功能
and some little fix

Just-Prog committed:
更加激进的请求策略，风险肯定大了一截，剩下的我不好说

Just-Prog committed:
Update README.md

Just-Prog committed:
Update README.md

Just-Prog committed:
GPLv3

Just-Prog committed:
票证人展关系判断改善以及其他一些优化

Just-Prog committed:
Merge branch 'main' of https://github.com/Just-Prog/Bilibili_show_ticket_auto_order

Just-Prog committed:
尝试使用 bili-ticket-gt-python 库替换浏览器进行验证码自动化解决
*实验功能，有待确认稳定性
*打包时使用的对应库版本: 0.1.3

Just-Prog committed:
糊了一个库存检测，目前能用，祈祷API别大改
虽然但是你b的后端接口怎么一个saleFlag一个sale_flag的混用啊

Just-Prog committed:
重写登录脚本，尽力拜托 Selenium 依赖
*破坏性更新，用户凭据文件与之前版本不兼容

Just-Prog committed:
跟进bili_ticket_gt_python上游修改
*光这一个包就把打包exe大小扩大了80M

Just-Prog committed:
fix requirements.txt

Just-Prog committed:
update readme.md

Just-Prog committed:
update README.md
宇宙安全声明

Just-Prog committed:
新版登录器 / 定时购票功能 (beta)

Just-Prog committed:
修复未付款订单的报错报错的bug

Just-Prog committed:
Create LICENSE.md

Anti CSDN License

Just-Prog committed:
Merge branch 'auto_captcha_solver'
引入极验验证码自动化解决方案

Just-Prog committed:
Merge branch 'main' of https://github.com/Just-Prog/Bilibili_show_ticket_auto_order

Just-Prog committed:
修复部分低版本系统无法正常显示unicode二维码的问题

Just-Prog committed:
暂时关闭订单创建前库存检测的操作
接到多条反馈提示 getV2 API 严重风控，临时禁用此方法

Just-Prog committed:
修复单身份不可同时购买多场次的问题 (beta) @mikumifa @teeedy
表现为：报错"指定的购买人存在已付款订单"
https://github.com/mikumifa/biliTickerBuy/commit/afb8c9b519f2b503ac0a40760ad68b0299b9b0c2
https://github.com/mikumifa/biliTickerBuy/commit/bc070250b5a6eb4f87c25985b98902cb4032fa75
感谢 @mikumifa @Teeedy

Just-Prog committed:
测试：自动处理极验验证码验证失败的bug

Just-Prog committed:
修缮极验错误处理措施 & 部分时间采集点采用阿里云授时服务器

Just-Prog committed:
将手机补全式验证需要的手机号迁移至配置文件实现预填写

123485k committed:
Update requirements.txt

添加了必要的ntplib依赖

Just-Prog committed:
Merge pull request #31 from 123485k/patch-3

Update requirements.txt

Just-Prog committed:
修复了少写一行token引用导致手机不全验证失败的重大bug

Just-Prog committed:
Merge branch 'main' of https://github.com/Just-Prog/Bilibili_show_ticket_auto_order

Just-Prog committed:
移除NTP授时
有很大概率在NTP这步因为连不到授时服务器把程序拖烂

Just-Prog committed:
添加主程序图标

Just-Prog committed:
提示当前导入的用户Cookie UID

Just-Prog committed:
打包指南

Just-Prog committed:
更新图标

Just-Prog committed:
传一下pyinstaller需要的spec文件

Just-Prog committed:
联票/纸质票电子票混合判断

Just-Prog committed:
fix: 有邮费的情况下撞到风控崩溃

Just-Prog committed:
Update README.md

Just-Prog committed:
fix: 部分接口接受数据类型限制变动导致基础功能受限
验证码之类的接口还没试过变没变

stmluyuer commented on main.py:
```diff
@@ -1,21 +1,31 @@
 specificID = None if a[1].split("=")[1].strip() == "None" else a[1].split("=")[1].strip()
 sleep = eval(a[2].split("=")[1].strip())
 token = None if a[3].split("=")[1].strip() == "None" else a[3].split("=")[1].strip()
+phone = None if a[4].split("=")[1].strip() == "None" else a[4].split("=")[1].strip()
```
新添加

Just-Prog committed:
修改程序关闭方案

Just-Prog committed:
更换网络请求实现 & 修复实名认证相关数据格式变化导致的报错

Just-Prog committed:
优化请求报错提示 & 启动时生成deviceId（不知道管用不管用）

Just-Prog committed:
关于BW/BML抢票行为的提前警告

Just-Prog commented:
暂关
